x/permission/keeper/keeper.go

```diff
@@ -73,17 +74,40 @@ func (k Keeper) AddGroupMember(ctx sdk.Context, groupID math.Uint, member sdk.Ac
 	id := k.groupMemberSeq.NextVal(store)
 	store.Set(memberKey, id.Bytes())
 	store.Set(types.GetGroupMemberByIDKey(id), k.cdc.MustMarshal(&groupMember))
+
+	// add to expire queue
+	if expiration != nil {
+		store.Set(types.GroupMemberPrefixQueue(expiration, types.GetGroupMemberByIDKey(id)[1:]), []byte{})
```

Comment:
use `memberID.Bytes()` directly instead of `GetGroupMemberByIDKey `?

Reply:
Done